src/entropic/greenkhorn.jl

```diff
@@ -0,0 +1,114 @@
+# Greenkhorn is a greedy version of the Sinkhorn algorithm
+# This method is from https://arxiv.org/pdf/1705.09634.pdf
+# Code is based on implementation from package POT
```

Comment:
It would be helpful to describe what the differences are (if there are any apart from implementation details).

Reply:
Yeah, there are. The paper implementation is actually just a couple of lines commented out. I'll point out in the code.

Reply:
Just realized that it's already in the code. Inside the `step!` function.